samples/snippets/src/main/java/com/example/bigquerystorage/WriteToDefaultStream.java

```diff
@@ -183,6 +194,7 @@ public void initialize(TableName parentTable)
               // column, apply the default value to the missing value field.
               .setDefaultMissingValueInterpretation(
                   AppendRowsRequest.MissingValueInterpretation.DEFAULT_VALUE)
+              .setRetrySettings(retrySettings)
```

Comment:
You should remove the retry code down on Line 263. Also check or other writers.

Reply:
Ack.  Removed the retry logic, modified writers in the other samples, and added a comment to each RetrySettings creation.